cargo test --features="bench"    fails to build

````
 cargo test --features="bench" 

   Compiling alacritty v0.1.0 (file:///home/matthias/alacritty)
error[E0061]: this function takes 3 parameters but 2 parameters were supplied
    --> src/term/mod.rs:2103:33
     |
990  | /     pub fn renderable_cells<'b>(
991  | |         &'b self,
992  | |         config: &'b Config,
993  | |         selection: Option<&'b Selection>,
...    |
1012 | |         )
1013 | |     }
     | |_____- defined here
...
2103 |               let iter = terminal.renderable_cells(&config, None);
     |                                   ^^^^^^^^^^^^^^^^ expected 3 parameters

error: aborting due to previous error

````

@ f298ebb2179eff947cdb9cb7a4bd96e9739eac5c

--- a/src/term/mod.rs
+++ b/src/term/mod.rs
@@ -2100,7 +2100,7 @@ mod benches {
         mem::swap(&mut terminal.grid, &mut grid);
 
         b.iter(|| {
-            let iter = terminal.renderable_cells(&config, None);
+            let iter = terminal.renderable_cells(&config, None, false);
             for cell in iter {
                 test::black_box(cell);
             }

